src/main/kotlin/com/openlattice/chronicle/services/candidates/CandidatesService.kt

```diff
@@ -0,0 +1,46 @@
+package com.openlattice.chronicle.services.candidates
+
+import com.openlattice.chronicle.candidates.Candidate
+import com.openlattice.chronicle.storage.ChroniclePostgresTables.Companion.CANDIDATES
+import com.openlattice.chronicle.storage.PostgresColumns.Companion.DELETE_ME
+import com.openlattice.chronicle.storage.StorageResolver
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.sql.Connection
+import java.time.LocalDate
+
+@Service
+class CandidatesService(
+    private val storageResolver: StorageResolver,
+) : CandidatesManager {
+
+    companion object {
+        private val logger = LoggerFactory.getLogger(CandidatesService::class.java)
+
+        private val CANDIDATES_COLUMNS = CANDIDATES
+            .columns
+            .subtract(setOf(DELETE_ME))
+
+        private val INSERT_CANDIDATE_SQL = """
+            INSERT INTO ${CANDIDATES.name} (${CANDIDATES_COLUMNS.joinToString { it.name }}) 
+            VALUES (${CANDIDATES_COLUMNS.map { "?" }.joinToString()})
+        """.trimIndent()
+    }
+
+    override fun registerCandidate(connection: Connection, candidate: Candidate) {
+        insertCandidate(connection, candidate)
```

Comment:
It seems reasonable to say candidates can only edit their own profiles, so there's no real need to make them securable objects. Just making sure this is deliberate.

Reply:
I don't understand ... candidates don't have accounts, they can't login, so they won't be able to edit their own profiles.

The choice to not make `Candidate` a securable object is deliberate but my reasoning is because it doesn't seem like an object users will need to set permissions on.